rtos/pulpos/pulp_archi/include/archi/chips/pulp/memory_map.h

```diff
@@ -119,11 +119,13 @@
 
 #define ARCHI_EU_DEMUX_OFFSET                 ( 0x00000 )
 #define ARCHI_MCHAN_DEMUX_OFFSET              ( 0x00400 )
+#define ARCHI_IDMA_DEMUX_OFFSET               ( 0x00400 )
 
 
 #define ARCHI_DEMUX_PERIPHERALS_ADDR          ( ARCHI_CLUSTER_ADDR + ARCHI_DEMUX_PERIPHERALS_OFFSET )
 
 #define ARCHI_EU_DEMUX_ADDR                   ( ARCHI_DEMUX_PERIPHERALS_ADDR + ARCHI_EU_DEMUX_OFFSET )
 #define ARCHI_MCHAN_DEMUX_ADDR                ( ARCHI_DEMUX_PERIPHERALS_ADDR + ARCHI_MCHAN_DEMUX_OFFSET )
+#define ARCHI_IDMA_DEMUX_ADDR                 ( ARCHI_DEMUX_PERIPHERALS_ADDR + ARCHI_IDMA_DEMUX_OFFSET )
```

Comment:
Similarly as above, here I would go for something like:
```suggestion
#define ARCHI_IDMA_DEMUX_ADDR                ARCHI_MCHAN_DEMUX_ADDR
```